vj4/handler/base.py

```diff
@@ -502,13 +503,16 @@ def decorate(coro):
 def sanitize(func):
   @functools.wraps(func)
   def wrapped(self, **kwargs):
+    accept_kwargs = inspect.getfullargspec(func).varkw != None
```

Comment:
unknown argument should always be ignored, even when `kwargs` is present.

Reply:
https://github.com/vijos/vj4/blob/master/vj4/handler/home.py#L135 uses kwargs

Reply:
and it does not use `sanitize`

Reply:
Great.